tests/test_nodes.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+"""Test the /nodes endpoint"""
+
+
+import io
+
+
+def test_create_dict(client, authenticate):  # pylint: disable=unused-argument
+    """Test creating a new dict."""
+    response = client.post(
+        "/nodes",
+        json={
+            "node_type": "data.dict.Dict.|",
+            "attributes": {"x": 1, "y": 2},
+            "label": "test_dict",
```

Comment:
please add a test that checks you get an error if you provide attributes that cannot be set by the user (e.g. uuid, mtime, ...)

Reply:
have you added it? (sorry if I'm overlooking it)